masonry_core/src/passes/event.rs

```diff
@@ -352,6 +352,10 @@ pub(crate) fn run_on_access_event_pass(
         false,
         |widget, ctx, props, event| {
             widget.on_access_event(ctx, props, event);
+            if !ctx.is_handled() && event.action == accesskit::Action::ScrollIntoView {
+                ctx.request_scroll_to_this();
+                ctx.set_handled();
+            }
```

Comment:
I think that this should move into the same block as `// handle focus events` below. This way, any parent can handle a bubbling `ScrollIntoView`, not just the direct target.